packages/patterns/src/components/patterns-manage-button.js

```diff
@@ -103,7 +103,7 @@ function PatternsManageButton( { clientId } ) {
 						}
 					} }
 				>
-					{ __( 'Detach' ) }
```

Comment:
There are also some e2e tests that wil need to be updated. You can look for `'Detach` there and update accordingly.

Reply:
Just updated! Can you re-review?